en/style/typography.md

```diff
@@ -30,42 +34,64 @@ These styles also determine the type of text element and typography class that w
   - `igx-typography__caption` for the caption typography style.
   - `igx-typography__overline` for the overline typography style.
```

Comment:
If we add the missing text styles I mentioned above, we need to check the implementation about how they define these styles.